kitsune/tidings/tasks.py

```diff
@@ -1,13 +1,14 @@
 from celery import shared_task
+from django.contrib.auth import get_user_model
 
 from kitsune.tidings.models import Watch
 
 
 @shared_task
-def claim_watches(user):
```

Comment:
We could also just delete this file `kitsune/tidings/tasks.py`, since it's not used anywhere as far as I can tell.